core/src/main/java/org/apache/carbondata/core/indexstore/DataMapStoreManager.java

```diff
@@ -30,7 +31,7 @@
 
   private static DataMapStoreManager instance = new DataMapStoreManager();
 
-  private Map<DataMapType, Map<String, TableDataMap>> dataMapMappping = new HashMap<>();
+  private Map<DataMapType, Map<String, AbstractTableDataMap>> dataMapMappping = new HashMap<>();
```

Comment:
I feel it is better to keep `Map<TableIdentifier, Map<Class, TableDataMap>>`, thus client can get all data map for one table easily.